chips/apollo3/src/lib.rs

```diff
@@ -90,7 +90,7 @@ pub unsafe fn init() {
     tock_rt0::zero_bss(&mut _szero, &mut _ezero);
 
     // Enable the cache controller
-    *(0x40018000i32 as *mut u32) = 1 | (1 << 10) | (1 << 20);
+    *(0x40018000i32 as *mut u32) = 1 | (1 << 10) | (1 << 20) | (1 << 8) | (1 << 9);
```

Comment:
Would it make sense to take this opportunity to convert this to the register interface? The setting of a series of non-sequential, unlabeled bits is a bit of a red flag :/

Reply:
Fixed